charts/openfaas-cloud/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for openfaas-cloud.
+# This is a YAML-formatted file.
+
+## ofBuilder is the section for of-builder configuration specific settings
+ofBuilder:
+
+  ## names is the name of the deployment for the of-builder component
+  name: of-builder
+
+  ## image is the container image that is used for the deployment of of-builder
+  image: openfaas/of-builder
+
+  ## tag is the image tag to be deployed when deploying this component
+  tag: 0.7.2
+
+  ## buildKitImage is the buildkit container image that is used within the deployment for building containers
+  buildKitImage: moby/buildkit
+
+  ## buildKitTag is the specific tagged image used for the buildkit container
+  buildKitTag: v0.6.2
+
+  ## replicaCount is the number of pod replicas for the the deployment
+  replicaCount: 1
+
+## edgeAuth the section for edge-auth configuration specific settings
+edgeAuth:
+
+  ## names is the name of the deployment for the edge-auth component
+  name: edge-auth
+
+  ## image is the container image that is used for the deployment of edge-auth
+  image: openfaas/edge-auth
+
+  ## tag is the image tag to be deployed when deploying this component
+  tag: 0.6.2
+
+  ## replicaCount is the number of pod replicas for the the deployment
+  replicaCount: 1
+
+  ## OAuthProvider is either github or gitlab depending on who is providing OAuth for your cluster
+  OAuthProvider: github
+
+  ## OAuthProviderBaseURL can be left blank for github, Gitlab users need to provide the url for their gitlab instance
+  OAuthProviderBaseURL:
+
+  ## clientId is generated by your Oauth App, and needs to be set here to connect correctly
+  clientId:
+
+  ## writeDebug is a setting to tell edge-auth to write debug messages
+  writeDebug: false
+
+  ## clientSecret is the value from yur OAuth application
+  clientSecret:
+
+## edgeRouter
+edgeRouter:
+
+  ## name is the name of edge-router component
+  name: edge-router
+
+  ## container is the edge-router container
+  container: openfaas/edge-router
+
+  ## tag is the tag for edge-router deployment
+  tag: 0.7.4
+
+## global values that are re-used across multiple components
+global:
+
+  ## rootDomain is the root domain of the ofc installation
+  rootDomain:
```

Comment:
Would it be a good idea to have example.com, or not?

Reply:
it wont work either way. We could have it in there as an ... example ... or we could add that info to the comment.

I am trying to work out a way to used the `required` functionality, but I cant get it to work with the printf functionality. That way, if its not set by the user the deployment fails.